src/.vuepress/theme/Sidebar.vue

```diff
@@ -201,6 +212,33 @@ export default {
         this.$refs.scrollwrap.scrollTop = activeLink.offsetTop - 50;
       }
     }, 500);
+
+    let item__1 = JSON.parse(localStorage.getItem('item__1'));
```

Comment:
Why don't you use a library or wrap yourself the `localStorage API` ? Code will be cleaner instead of stringify / parse at each item set / get